backend/queries/time.js

```diff
@@ -54,10 +45,20 @@ const deleteHoursByVolunteerId = async (id, promise) => {
   return await db.any(deleteQuery, id);
 }
 
+const updateVolunteerHours = async (v_id, event_duration) => {
+  const updateQuery = `
+  UPDATE volunteers_hours 
+  SET banked_time = $/event_duration/ 
+  WHERE volunteer_id = $/v_id/  
+  RETURNING *
+  `
+  return await db.one(updateQuery, {id:v_id, number:event_duration});
+}
 /* EXPORT */
 module.exports = {
     allBankedHours,
     allPlannedHours,
```

Comment:
Since you removed the `allPlannedHours` function, it also should me removed from here

Reply:
the Array_agg is in the query doc and the allPlannedHours hours are removed. I forgot to push that change last night.